Podcast/SubscribeButton.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  SubscribeButton.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 2/18/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SubscribeButton: UIButton {
```

Comment:
This class seems unnecessary, we should just have two button images, one for each selected state